JASP-Engine/JASP/R/common.R

```diff
@@ -754,6 +755,28 @@ callback <- function(results=NULL, progress=NULL) {
 	stop("could not clean value")
 }
 
+# .clean <- function(value) {
```

Comment:
Can you remove these commented out lines? We can update .clean at some point, but this seems unnecessary :)